astroid/nodes/node_classes.py

```diff
@@ -478,7 +487,7 @@ def __init__(
             parent=parent,
         )
 
-    assigned_stmts: AssignedStmtsCall["AssignName"]
+    assigned_stmts: ClassVar[AssignedStmtsCall["AssignName"]]
```

Comment:
Fyi, I just thought that we could included `ClassVar` in the type alias, too. However, that is a type error. `ClassVar` is only accepted on class attributes.